src/RepositoryApiClientHandler.cs

```diff
@@ -38,6 +38,7 @@ private async Task<HttpResponseMessage> SendExAsync(HttpRequestMessage request,
                 _baseAddress = GetBaseAddress(beforeSendResult.RegionalDomain);
             }
             request.RequestUri = new Uri(_baseAddress, request.RequestUri.PathAndQuery);
+            request.Headers.Add("Accept-Encoding", "gzip");
```

Comment:
Instead of adding the header here, I was thinking if it would be a good idea to add it to the `DefaultRequestHeaders` when we create the http client. This will allow users to remove/customize the `Accept-Encoding` header, if needed.
https://github.com/Laserfiche/lf-repository-api-client-dotnet/blob/0a2f4f45413baa4eb056b138928bb7b72955ce91/src/RepositoryApiClient.cs#LL50C37-L50C37

Reply:
ok i added this implementation if it is ok

` if (_httpClient != null)
  {
       _httpClient.DefaultRequestHeaders.Add("Accept-Encoding", "gzip");
}`

Reply:
thanks, can you also make a similar change in the js repo client PR?